Destiny.ScrimTracker.Api/Controllers/GuardianController.cs

```diff
@@ -5,55 +5,90 @@
 using System.Threading.Tasks;
 using Destiny.ScrimTracker.Api.Requests;
 using Destiny.ScrimTracker.Logic.Models;
+using Destiny.ScrimTracker.Logic.Repositories;
 using Destiny.ScrimTracker.Logic.Services;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
 
 namespace Destiny.ScrimTracker.Api.Controllers

```

Comment:
Can you help me understand the `Api` portion of this namespace? Now that this is returning `View()` I'm assuming its doing full on HTML rendering, which isn't an API in the classic sense. If I'm understanding it correctly, should this namespace instead be `App`?

Reply:
Yeaaaaa. That's a leftover relic from when this was an actual API. Will change now that it isn't

Reply:
No need to change this now but maybe in a future PR.

Reply:
This'll get addressed in the next PR. That's the main reason it's 50+ files